pedsnetdcc/check_fact_relationship.py

```diff
@@ -0,0 +1,283 @@
+                'field': 'fact_id_1',
+                'sample': sample
+            })
+
+        if results['bad_fr_obs_id2_sample']['data']:
+            names = results['bad_fr_obs_id2_sample']['field_names']
+            data = results['bad_fr_obs_id2_sample']['data']
+            sample = dict(zip(names, data))
+            logger.warn({
+                'msg': 'Fact relationship table has bad observation refs.',
+                'field': 'fact_id_2',
+                'sample': sample
+            })
+
+        if not results['bad_fr_obs_id1_sample']['data'] and \
+                not results['bad_fr_obs_id2_sample']['data']:
+            logger.info({
+                'msg': 'Fact relationship table observation refs are valid.'
+            })
+
+        if results['bad_fr_meas_id1_sample']['data']:
+            names = results['bad_fr_meas_id1_sample']['field_names']
+            data = results['bad_fr_meas_id1_sample']['data']
+            sample = dict(zip(names, data))
+            logger.warn({
+                'msg': 'Fact relationship table has bad measurement refs.',
+                'field': 'fact_id_1',
+                'sample': sample
+            })
+
+        if results['bad_fr_meas_id2_sample']['data']:
+            names = results['bad_fr_meas_id2_sample']['field_names']
+            data = results['bad_fr_meas_id2_sample']['data']
+            sample = dict(zip(names, data))
+            logger.warn({
+                'msg': 'Fact relationship table has bad measurement refs.',
+                'field': 'fact_id_2',
+                'sample': sample
+            })
+
+        if not results['bad_fr_meas_id1_sample']['data'] and \
+                not results['bad_fr_meas_id2_sample']['data']:
+            logger.info({
+                'msg': 'Fact relationship table measurement refs are valid.'
+            })
+
+        if results['bad_fr_visit_id1_sample']['data']:
+            names = results['bad_fr_visit_id1_sample']['field_names']
+            data = results['bad_fr_visit_id1_sample']['data']
+            sample = dict(zip(names, data))
+            logger.warn({
+                'msg': 'Fact relationship table has bad visit refs.',
+                'field': 'fact_id_1',
+                'sample': sample
+            })
+
+        if results['bad_fr_visit_id2_sample']['data']:
+            names = results['bad_fr_visit_id2_sample']['field_names']
+            data = results['bad_fr_visit_id2_sample']['data']
+            sample = dict(zip(names, data))
+            logger.warn({
+                'msg': 'Fact relationship table has bad visit refs.',
+                'field': 'fact_id_2',
+                'sample': sample
+            })
+
+        if not results['bad_fr_visit_id1_sample']['data'] and \
+                not results['bad_fr_visit_id2_sample']['data']:
+            logger.info({
+                'msg': 'Fact relationship table visit refs are valid.'
+            })
```

Comment:
How about having our commands return an OS status code of 0 when there is no error, and a positive value when there is an error.